packages/api/endpoints/providers.js

```diff
@@ -53,18 +54,21 @@ async function list(req, res) {
  */
 async function get(req, res) {
   const id = req.params.id;
+  const knex = await getKnexClient({ env: process.env });
+  const providerPgModel = new ProviderPgModel();
 
-  const providerModel = new Provider();
   let result;
   try {
-    result = await providerModel.get({ id });
+    const providerRecord = await providerPgModel.get(knex, { name: id });
+    result = translatePostgresProviderToApiProvider(providerRecord);
   } catch (error) {
-    if (error instanceof RecordDoesNotExist) return res.boom.notFound('Provider not found.');
+    if (error instanceof RecordDoesNotExist) return res.boom.notFound(`Provider ${JSON.stringify(id)} not found.`);
```

Comment:
Do we need to stringify id?

Reply:
No - sometimes it makes sense to do so defensively, but in this case the param should always be a string.   f4547c9c5